selfdrive/car/ford/carstate.py

```diff
@@ -131,6 +132,10 @@ def get_can_parser(CP):
       messages += [
         ("Lane_Assist_Data3_FD1", 33),
       ]
+    else:
+      messages += [
+        ("INSTRUMENT_PANEL", 10),
```

Comment:
are you sure this is 10Hz? It's 1Hz on our Bronco, and takes 0.48s to show up on the process replay route (`VehicleOperatingModes` is instant for example), which makes sense given it's 1Hz in that route. When it shows up is when carState now goes valid, also making sense.

Reply:
oh, it was an oversight from me. must be 1Hz

Reply:
Turns out it's not even 1Hz! sometimes the message is completely dropped
![image](https://github.com/commaai/openpilot/assets/4038174/0244993d-47a0-476c-8bc4-3c20e842d6d0)

When I look at it in plot juggler it looks fine though...
![image](https://github.com/commaai/openpilot/assets/4038174/c11c4235-e4b7-49c0-9052-a9742e7a22f1)

Reply:
for some reason it's still invalid when freq=1

Reply:
See

> That's because it's just expanding the invalid carState time/messages out to when we see all the CAN messages, it takes 0.79 seconds for the TOYOTA3 route to go valid for example

Reply:
Oh I thought it would go valid quicker for 1hz than 10hz

Reply:
It goes valid when it sees the message